src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -58,6 +58,25 @@ def setup_accelerator(tool_button):
 
 
 class ToggleToolButton(Gtk.ToggleToolButton):
+    '''
+    UI for toggletoolbutton.
+    A ToggleToolButton is a ToolItem that contains a toggle button,
+    having an icon, a tooltip palette, and an accelerator.
+    Use ToggleToolButton.new() to create a new ToggleToolButton.
+
+    Fuctions:
+        ToggleToolButton()
+            Returns: a new toggletoolbutton
```

Comment:
You don't need the Functions section.  - this docstring will show up under the constructor for the toggle toolbutton anyway.